tests/unit-tests/UserMergeScenariosTests.swift

```diff
@@ -144,7 +145,9 @@ class UserMergeScenariosTests: XCTestCase, AuthProvider {
                    XCTFail("Expected events to be logged but found nil")
                }
         
-        IterableAPI.setUserId("testuser123", merge: true)
+        let identityResolution = IterableIdentityResolution(replayOnVisitorToKnown: true, mergeOnAnonymousToKnown: true)
```

Comment:
Here, we should not need a identityResolution object as the default of `true` `true` should be used.

Reply:
Gotcha I will make the change